scripts/custom_lint_checks_test.py

```diff
@@ -0,0 +1,80 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For details on how to write such tests, please refer to
+# https://github.com/oppia/oppia/wiki/Writing-Tests-For-Pylint
+
+import os
+import sys
+import unittest
+
+_PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+_PYLINT_PATH = os.path.join(_PARENT_DIR, 'oppia_tools', 'pylint-1.7.1')
+sys.path.insert(0, _PYLINT_PATH)
+
+# Since these module needs to be imported after adding Pylint path,
+# we need to disable isort for the below lines to prevent import
+# order errors.
+# pylint: disable=wrong-import-position
+# pylint: disable=relative-import
+import astroid  # isort:skip
+import custom_lint_checks  # isort:skip
+import pylint  # isort:skip
+from pylint.testutils import CheckerTestCase  # isort:skip
```

Comment:
from pylint import testutils

Then in code, use testutils.CheckerTestCase. Never import classes directly.

(We should have a lint check for that :) )

Reply:
Done! Thanks :) 

(I'll add this to the list of custom checkers.)